lstchain/tests/test_lstchain.py

```diff
@@ -15,6 +15,7 @@
 test_data = Path(os.getenv('LSTCHAIN_TEST_DATA', 'test_data'))
 test_r0_path = test_data / 'real/R0/20200218/LST-1.1.Run02008.0000_first50.fits.fz'
 test_r0_path2 = test_data / 'real/R0/20200218/LST-1.1.Run02008.0100_first50.fits.fz'
+test_r0_path3 = test_data / 'real/R0/20200218/LST-1.1.Run02008.0201.fits.fz'
```

Comment:
Does this file actually exist?

Reply:
Yes. I checked and it ran successfully.

Reply:
This is one of the initial small files. Why add the third here? We should only use the `first_50` file I think.

Reply:
Just downloaded again the test dataset and I cannot find it. And there are already two dl1/dl2 files that can be used.

Reply:
I need some file to use create a fixture in `conftest.py`, so that I can use it `test_tools.py`. Can you give a suggestion on it? I suppose I have the file in my local test_data and didn't confirm before whether it was used here or not.

Reply:
@morcuended you are right, that file is not on the data server

Reply:
Looking at the file list, we should also probably add the 2006 first 50.

Reply:
I suggest moving all the fixtures created in `lstchain/scripts/tests/test_lstchain_scripts.py` to `conftest.py`

Reply:
If moving all the fixtures created in `lstchain/scripts/tests/test_lstchain_scripts.py` to `lstchain/conftest.py` is an agreeable solution, maybe it can be done in another smaller/quicker PR.

Reply:
done in #640